index.html

```diff
@@ -0,0 +1,124 @@
+            alt="Trumpet image"
+            width="160"
+            height="70"
+          />
+        </div>
+        <nav id="nav-bar">
+          <ul>
+            <li><a class="nav-link" href="#something">Features</a></li>
+            <li><a class="nav-link" href="#somethingmore">How It Works</a></li>
+            <li><a class="nav-link" href="#evenmore">Prices</a></li>
+          </ul>
+        </nav>
+      </header>
+      <div>
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+          <input
+            name="email"
+            id="email"
+            type="email"
+            placeholder="Enter your email address"
+            required=""
+          />
+          <input name="submit" id="submit" type="submit" value="Submit" />
+        </form>
+      </div>
+
+      <section id="something">
+        <div>
+          <div class="icon">
+            <img
+              src="https://cdn-icons-png.flaticon.com/512/1350/1350727.png"
+              alt="Materials"
+              width="60"
+              height="70"
+            />
+          </div>
+          <div class="descrip">
+            <h2>Premium Materials</h2>
+            <p>
+              Our trumpets use the shiniest brass which is sourced locally. This
+              will increase the longevity of your purchase.
+            </p>
+          </div>
+        </div>
+        <div>
+          <div class="icon">
+            <img
+              src="https://cdn-icons-png.flaticon.com/512/5770/5770775.png"
+              alt="Materials"
+              width="70"
+              height="70"
+            />
+          </div>
+
+          <div class="descrip">
+            <h2>Fast Shipping</h2>
+            <p>
+              We make sure you recieve your trumpet as soon as we have finished
+              making it. We also provide free returns if you are not satisfied.
+            </p>
+          </div>
+        </div>
+        <div>
+          <div class="icon">
+            <img
+              src="https://cdn-icons-png.flaticon.com/512/3500/3500954.png"
+              alt="Materials"
+              width="60"
+              height="60"
+            />
+          </div>
+          <div class="descrip">
+            <h2>Quality Assurance</h2>
+            <p>
+              For every purchase you make, we will ensure there are no damages
+              or faults and we will check and test the pitch of your instrument.
+            </p>
+          </div>
+        </div>
+      </section>
+      <section id="somethingmore">
+        <div>
+          <iframe
+            width="420"
+            height="315"
+            src="https://www.youtube.com/embed/nODd-8rvJJU"
+          >
+          </iframe>
+        </div>
+      </section>
+      <section id="evenmore">
+        <div>Price1</div>
+        <div>Price2</div>
+        <div>Price3</div>
+      </section>
```

Comment:
Change the id and section names to something more descriptive

Reply:
**_Change the id names to something more descriptive_**

  

- [ ] I think it would be better to make it a bit clear with the specific `id` being referred to. The `id` word could be any word in the code so I suggest being specific by saying the `id` selector inside the `section` tag.
- [ ] I suggest the comment should be more politely written by saying **_"I think it would be better ...."_**